cmd/gpu_fakedev/gpu_fakedev.go

```diff
@@ -289,6 +293,85 @@ func generateDriFiles(opts genOptions) {
 		}
 	}
 	log.Printf("Done, created %d dirs, %d devices and %d files.", opts.dirs, opts.devs, opts.files)
+
+	makeXelinkSideCar(opts)
+}
+
+func makeXelinkSideCar(opts genOptions) {
+	topology := opts.Capabilities["connection-topology"]
+	gpus := opts.DevCount
+	tiles := opts.TilesPerDev
+	connections := opts.Capabilities["connections"]
+
+	if topology != fullyConnected {
+		saveSideCarFile(connections)
+	} else {
+		saveSideCarFile(buildConnectionList(gpus, tiles))
+	}
+	log.Printf("XELINK: generate xelink sidecar label file, using (GPUs: %d, Tiles: %d, Topology: %s)", gpus, tiles, topology)
```

Comment:
As this comes after label file is already generated, it could use past tense, similarly to other log messages:
```suggestion
	log.Printf("XELINK: generated xelink sidecar label file using (GPUs: %d, Tiles: %d, Topology: %s)", gpus, tiles, topology)
```